--- node_modules/react-native/scripts/react-native-xcode.sh	2018-05-09 06:21:10.000000000 +0900
+++ node_modules/react-native/scripts/react-native-xcode.sh.patched	2018-07-11 20:12:20.720123351 +0900
@@ -92,14 +92,7 @@
 set -x
 DEST=$CONFIGURATION_BUILD_DIR/$UNLOCALIZED_RESOURCES_FOLDER_PATH
 
-if [[ "$CONFIGURATION" = "Debug" && ! "$PLATFORM_NAME" == *simulator ]]; then
-  IP=$(ipconfig getifaddr en0)
-  if [ -z "$IP" ]; then
-    IP=$(ifconfig | grep 'inet ' | grep -v ' 127.' | cut -d\   -f2  | awk 'NR==1{print $1}')
-  fi
-
-  echo "$IP" > "$DEST/ip.txt"
-fi
+echo "%%PACKAGER_IP%%" > "$DEST/ip.txt"
 
 BUNDLE_FILE="$DEST/main.jsbundle"
 
